spring-kafka/src/main/java/org/springframework/kafka/support/serializer/DelegatingByTopicDeserializer.java

```diff
@@ -75,4 +78,9 @@ public Object deserialize(String topic, Headers headers, byte[] data) {
 		return findDelegate(topic).deserialize(topic, headers, data);
 	}
 
+	@Override
```

Comment:
Need the `since` tag.

Reply:
No need: this is an impl of existing API.
Technically it was there even before, but relied on a `default` method.

So, all good here.